src/usb.c

```diff
@@ -812,7 +812,7 @@ handle_create_session (XdpDbusUsb            *object,
                            &options_builder,
```

Comment:
Does it become possible to replace the bespoke option filtering implementation in the USB portal's `filter_access_devices` function with the standard `xdp_filter_options` now that `xdp_filter_options` has a userdata pointer?